src/System.IO.FileSystem/src/System/IO/FileSystemInfo.cs

```diff
@@ -8,11 +8,13 @@
 using Microsoft.Win32;
 using System.Text;
 using System.Runtime.InteropServices;
+using System.Runtime.Serialization;
 using System.Runtime.Versioning;
 
 namespace System.IO
 {
-    public abstract partial class FileSystemInfo
+    [Serializable]
```

Comment:
Thanks. I MSDN is ambigously worded.

---
In reply to: [83738419](https://github.com/dotnet/corefx/pull/12736#discussion_r83738419) [](ancestors = 83738419)